### PR TITLE
Feature: Dashboard error states

### PR DESCRIPTION
**Pull request by rohjs**
## Description

This PR is to show error tooltips when it fails to fetch the total staked balance || APRs

- [x] Create APIs:
  - [x] `/api/totalStaked`: read `totalStaked` balance from the staking contract
  - [x] `/api/calcApr`: calculate APR of reward tokens  
- [x] Show `AprFallback` when it fails to fetch the `totalStaked` balance
- [x] Show `AprItemFallback` when it fails to fetch the calculated APRs  

## Screenshots

**▼ When failed to load `totalStaked`**

https://github.com/planetarium/wncg-staking-interface/assets/19285811/b08f5170-3082-44b8-979c-3873e7a74976

**▼ When failed to load APRs**

https://github.com/planetarium/wncg-staking-interface/assets/19285811/2961e5eb-179c-4dc9-af5f-8e5514da6198

### CODE REVIEW

File: components/staking/Dashboard/AprFallback.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+          </Button>
+        </p>
+      </StyledStakingDashboardAprFallback>
+    )
+  }
+
+  return (
+    <StyledStakingDashboardAprFallback>
+      <StyledStakingDashboardApr className="aprList">
+        <div className="aprItem">
```
Comment (24pAs) [inline components/staking/Dashboard/AprFallback.tsx:45]: The `<dt>` and `<dd>` must always be wrapped in a `<dl>`.

File: components/staking/Dashboard/BALApr.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+            >
+              Harvest
+            </Button>
+          </motion.div>
+        )}
+      </AnimatePresence>
+    </div>
+  )
+
+  return (
```
Comment (24pAs) [inline components/staking/Dashboard/BALApr.tsx]: Can you explain this `return` features?

**Review by 24pAs — COMMENTED** [review]
I haven't use together `ErrorBoundary` and `Suspense`. It is so good. Thank you!

File: components/staking/Dashboard/AprFallback.tsx (hunk 1 above)
Comment (rohjs) [reply components/staking/Dashboard/AprFallback.tsx:45]: Oops! My mistake. Thanks for the comment 👍

File: components/staking/Dashboard/BALApr.tsx (hunk 2 above)
Comment (rohjs) [reply components/staking/Dashboard/BALApr.tsx]: I should have removed the code below. Thank you!

**Review by 24pAs — APPROVED** [review]

**Pull request merged**